examples/chat/Cargo.toml

```diff
@@ -4,12 +4,20 @@ version = "0.1.0"
 edition = "2024"
 
 [dependencies]
+bb8 = "0.9"
+bytes = "1"
+chrono = { version = "0.4", features = ["serde"] }
+cookie = { version = "0.18", features = ["signed", "private"] }
+diesel = { features = ["chrono", "uuid"], version = "2" }
+diesel-async = { version = "0.7", features = ["bb8", "postgres"] }
+dotenvy = "0.15"
 http = "1"
 serde = "1"
 serde_json = "1"
 tokio = { version = "1", features = ["macros", "rt-multi-thread"] }
+uuid = { version = "1", features = ["v4", "serde"] }
```

Comment:
I'm going to implement a hash id wrapper on top of uuid so they don't leak into urls. There will be a FromStr and Deserialize impl to make the hashing opaque for everyone.